app/src/test/java/org/fossasia/openevent/app/presenter/AttendeePresenterTest.java

```diff
@@ -122,10 +125,12 @@ public void shouldLoadAttendeesSuccessfully() {
         attendeesPresenter.loadAttendees(false);
 
         inOrder.verify(attendeesView).showProgressBar(true);
+        inOrder.verify(attendeesView).showEmptyView(false);
```

Comment:
Remove the false cases and add them as separate tests. These tests have become very self dependent. Each test should handle only one aspect of the view.

Will simplify them later as this is a low priority issue